labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -99,7 +98,10 @@ object ListManipulationExercise01 {
    * Implement it whatever way suites you best. Hint: it can be done in a neat way using recursion. 
    */
   def tails[T](l: List[T]): List[List[T]] = {
-    error("fix me")
+    if (l == List())
+      List(List())
+    else
+      List(l) ++ tails(l.drop(1))
   }
 }
```

Comment:
It didn't occur to me that l.tail and l.drop(1) would produce the same result. I wasn't thinking about the methods in the same way, but your solution got me thinking in that mindset. Interesting!